app/Http/Controllers/UserController.php

```
@@ -0,0 +1,29 @@
+class UserController extends Controller
+{
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \App\User  $user
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request, User $user)
+    {
+        $validatedData = $request->validate([
```

PHPMD (2.15.0)

Avoid unused local variables such as '$validatedData'. (Unused Code Rules)

(UnusedLocalVariable)